doc/js_tutorials/js_assets/js_image_classification.html

```diff
@@ -0,0 +1,384 @@
+                    <tbody>
+                        <tr>
+                            <th scope="row">1</th>
+                            <td id="label0" align="center"></td>
+                            <td id="prob0" align="center"></td>
+                        </tr>
+                        <tr>
+                                <th scope="row">2</th>
+                                <td id="label1" align="center"></td>
+                                <td id="prob1" align="center"></td>
+                        </tr>
+                        <tr>
+                                <th scope="row">3</th>
+                                <td id="label2" align="center"></td>
+                                <td id="prob2" align="center"></td>
+                        </tr>
+                    </tbody>
+                </table>
+                <p id='time' align="left"></p>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <div class="caption">
+                    canvasInput <input type="file" id="fileInput" name="file" accept="image/*">
+                </div>
+            </td>
+            <td></td>
+        </tr>
+    </table>
+</div>
+
+<div>
+    <h3>Help function</h3>
+    <p>1.The parameters for model inference which you can modify to investigate more models.</p>
+    <textarea class="code" rows="15" cols="100" id="codeEditor" spellcheck="false"></textarea>
+    <p>2.Main loop in which will read the image from canvas and do inference once.</p>
+    <textarea class="code" rows="27" cols="100" id="codeEditor1" spellcheck="false"></textarea>
+    <p>3.Load labels from txt file and process it into an array.</p>
+    <textarea class="code" rows="9" cols="100" id="codeEditor2" spellcheck="false"></textarea>
+    <p>4.Get blob from image as input for net, and standardize it with <b>mean</b> and <b>std</b>.</p>
+    <textarea class="code" rows="11" cols="100" id="codeEditor3" spellcheck="false"></textarea>
+    <p>5.Fetch model from network and save to emscripten file system.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor4" spellcheck="false"></textarea>
+    <p>6.The post-processing, including softmax if needed and get the top classes from the output vector.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor5" spellcheck="false"></textarea>
+    <p class="err" id="errorMessage"></p>
+</div>
+
+<div id="appendix">
+    <h2>Model Info:</h2>
+</div>
+
+<script src="utils.js" type="text/javascript"></script>
+
+<script id="codeSnippet" type="text/code-snippet">
+inputSize = [224,224];
+mean = [104, 117, 123];
+std = 1;
+rgb = true;
+
+// record if need softmax function for post-processing
+needSoftmax = false; 
+
+// url for model, config and label files, can from native or Internet
+// for model type like onnx which doesn't need config file, just let configUrl = ""
+labelsUrl = "labels1000.txt";
+modelUrl = "bvlc_googlenet.caffemodel";
+configUrl = "bvlc_googlenet.prototxt";
```

Comment:
It means these model/label files should be hosted within the same path of the HTML/JS files. Is that the case for the opencv docs deployment? Any suggestions? @dkurt @alalek ?

Reply:
Seems like these are local paths of downloaded models from
```
 {
            "model": "googlenet",
            "mean": "104, 117, 123",
            "std": "1",
            "labels": "1000",
            "rgb": "true",
            "softmax": "no",
            "modelUrl": "http://dl.caffe.berkeleyvision.org/bvlc_googlenet.caffemodel",
            "configUrl": "https://raw.githubusercontent.com/BVLC/caffe/master/models/bvlc_googlenet/deploy.prototxt"
        },        
```

Reply:
Yes, the example can load model from native or by url. But some of the download urls will cause network issue when load model by url.